h1.c

```
@@ -0,0 +1,310 @@
+
+void putfont8(uint8_t x, uint8_t y, uint8_t* font) {
+  uint8_t i;
+  for(i = 0; i < 8; i++) {
+    putbyte(x, y + i, font[i]);
+  }
+}
+
+void putfont16(uint8_t x, uint8_t y, uint8_t* font) {
+  uint8_t i, p;
+  for(i = 0, p = 0; i < 16; i++) {
+    put2byte(x, y + i, font[p++], font[p++]);
```

cppcheck

[error] Expression 'x,y+i,font[p++],font[p++]' depends on order of evaluation of side effects

(unknownEvaluationOrder)